ui/keyboardMove/src/main.ts

```diff
@@ -136,6 +137,7 @@ export function ctrl(root: KeyboardMoveRootCtrl): KeyboardMove {
     checker: root.speakClock ? new KeyboardChecker() : undefined,
     opponent: root.data.opponent?.user?.username,
     speakClock: () => root.speakClock?.(),
+    goBerserk: () => root.goBerserk?.(),
```

Comment:
why the `?.()`? The type says the function is always available `goBerserk(): void;`

Should it be optional in the type instead?

Reply:
Really not sure about that. I just copied the speakClock from above. I would like to know how speakClock is optional and it is not. I need to learn more about these types.

Reply:
yeah looks like the code you copied has the same issue.